tests/integration/kubernetes/lib.sh

```diff
@@ -251,11 +250,11 @@ print_node_journal() {
 	shift 2
 	local img="quay.io/prometheus/busybox"
 
-	kubectl debug --image "$img" -q -i "node/${node}" \
-		-- chroot /host journalctl -x -t "$id" --no-pager "$@"
-	# Delete the debugger pod
-	kubectl get pods -o name | grep "node-debugger-${node}" | \
-		xargs kubectl delete > /dev/null
+	kubectl debug --image "$img" -q "node/${node}" -- chroot /host bash -c "sleep infinity" >&2
```

Comment:
I'm wondering why we never used the `exec_host()` here.... IIRC `print_node_journal()` was copied from CCv0 branch and `exec_host()` already existed on main (but not CCv0). Maybe we overlooked `exec_host()`?

I don't see any technical impeachment for using exec_host, expect that maybe we would need to move exec_host from tests_common.sh to lib.sh to avoid any cyclical dependency between the two libs. @BbolroC does it sound as a good idea to you?  --- Can be done in a separate PR.

Reply:
> This PR will be improved when https://github.com/kata-containers/kata-containers/pull/10232 is merged. Thanks!

Yes, that is why I mentioned it earlier. Once this PR is merged, all `kubectl debug` will be replaced with `exec_host()`, giving us a single point of control for that functionality. Thanks.